Gruntfile.js

```diff
@@ -8,7 +8,7 @@ module.exports = function(grunt) {
 
   gtx.config(gruntConfig);
 
-  gtx.alias('build', ['build-dist', 'less', 'copy']);
+  gtx.alias('build', ['build-dist', 'less', 'sass', 'copy']);
```

Comment:
My only concern here is that the SASS compiler, while interesting to include is necessary for the created of a standalone CSS file. And that it's also nothing to do with your button dropdown.

People who use SASS will compile the the SASS in their project, People who use LESS will compile that in their own projects. People who want a standalone CSS file will use whatever is generated, we only need 1 to be generated, might as well use the LESS one.